asab/metrics/service.py

```diff
@@ -45,6 +46,11 @@ def __init__(self, app, service_name):
 			if target_type == 'influxdb':
 				from .influxdb import MetricsInfluxDB
 				target = MetricsInfluxDB(self, 'asab:metrics:{}'.format(target))
+
+			if target_type == "prometheus":
+				from. prometheus import PrometheusTarget
+				target = PrometheusTarget(self, 'asab:metrics:prometheus')
+				self.PrometheusTarget = target
```

Comment:
`self.PrometheusTarget` should be set to `None` in the start of `__init__` - so that you may use that for detection if this target is available.